### PR TITLE
Add documentation for JAVA custom metrics + publish to maven

### PR DESCRIPTION
**Pull request by yotamloe**
Still TODO:
1. Review docs
2. Publish to maven

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,157 @@ This registry should be able to be used any application that uses micrometer for
...
-3) The registry class is `com.newrelic.telemetry.micrometer.NewRelicRegistry`. 
-It will need to be built via a `com.newrelic.telemetry.micrometer.NewRelicRegistryConfig` implementation. An example of how you might do that is in the wiki: [Spring-Config-Example](https://github.com/newrelic/micrometer-registry-newrelic/wiki/Spring-Config-Example)
+3) Quick start:
+   
+Replace the placeholders in the code (indicated by the double angle brackets `<< >>`) to match your specifics.
 
-In order to send metrics to New Logz, you will also need an Insert API Key. Please see [New Relic Api Keys](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#) for more information.
+| Environment variable | Description |Required/Default|
+|---|---|---|
+|listener-url|  The Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic. For more details, see the [Prometheus configuration file remote write reference. ](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) | Required|
```
Comment (Doron-Bargo) [inline README.md]: Why are we sending the customers to Prometheus documentation ?

Comment (yotamloe) [reply README.md]: You are correct ill change that

**Comment by yotamloe** [comment]
@YajB @Doron-Bargo The project now is approved and waiting for the sync to the maven central repository: [https://issues.sonatype.org/browse/OSSRH-64636](https://issues.sonatype.org/browse/OSSRH-64636)
Can you review the repo documentation so I can open a PR for our docs as soon as this project is on maven.

**Review by Doron-Bargo — APPROVED** [review]
Amazing  work

**Pull request merged**